homeassistant/components/switchbee/cover.py

```diff
@@ -0,0 +1,204 @@
+"""Support for SwitchBee cover."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from switchbee.api import (
+    SwitchBeeDeviceOfflineError,
+    SwitchBeeError,
+    SwitchBeeTokenError,
+)
+from switchbee.const import SomfyCommand
+from switchbee.device import DeviceType, SwitchBeeBaseDevice
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    CoverDeviceClass,
+    CoverEntity,
+    CoverEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import SwitchBeeCoordinator
+from .entity import SwitchBeeDeviceEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up SwitchBee switch."""
+    coordinator: SwitchBeeCoordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        SwitchBeeCover(device, coordinator)
+        for device in coordinator.data.values()
+        if device.type in [DeviceType.Shutter, DeviceType.Somfy]
+    )
+
+
+class SwitchBeeCover(SwitchBeeDeviceEntity, CoverEntity):
+    """Representation of an SwitchBee cover."""
+
+    def __init__(
```

Comment:
device_class is static, so it can be a class attribute:
```suggestion
class SwitchBeeCover(SwitchBeeDeviceEntity, CoverEntity):
    """Representation of an SwitchBee cover."""

    _attr_device_class = CoverDeviceClass.

    def __init__(
```